boost_library_usage_dashboard/management/commands/run_boost_library_usage_dashboard.py

```
@@ -0,0 +1,135 @@
+
+        self.stdout.write("Step 3: Rendering HTML files...")
+        render_dashboard_html(base_dir=settings.BASE_DIR, output_dir=output_dir)
+
+        self.stdout.write(
+            self.style.SUCCESS(f"Dashboard artifacts generated at: {output_dir}")
+        )
+
+        if options["publish"]:
+            target_repo = (options["target_repo"] or "").strip()
+            if not target_repo:
+                raise ValueError("--publish requires --target-repo")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)